modules/client-configuration/pages/delete-clients.adoc

```diff
@@ -19,13 +28,200 @@ For more information about the system set manager, see xref:client-configuration
 
 [IMPORTANT]
 ====
-It is not possible to automatically clean up a a traditional client after deleting it. You have to take care of this
-yourself. Furthermore cleaning up a Salt minion does not remove Salt itself.
+It is not possible to automatically clean up a traditional client after deleting it.
+You have to take care of this yourself.
+Furthermore, cleaning up a Salt client does not remove Salt itself.
+====
+
+[NOTE]
+====
+Normally you migrate a traditional client to a Salt client without deleting the client.
+Salt automatically detects that you have a traditional client and does the necessary changes itself.
+But if you already deleted the traditional client and want to register it as a Salt client again, see
+xref:administration:troubleshooting/tshoot-register-trad-as-salt-after-deletion.adoc[].
 ====
 
+
+
+== Delete a Salt Client on the Command Line (with API Call)
+
+.Procedure: Deleting a Client from the Server
+
+. Delete the client with the FQDN (Fully Qualified Domain Name):
++
+----
+spacecmd system_delete FQDN
+----
++
+[command]``spacecmd system_delete`` also deletes the Salt key.
+
+[command]``system_delete`` offers the following options:
+
+----
+usage: system_delete [options] <SYSTEMS>
+
+    options:
+          -c TYPE - Possible values:
+             *  'FAIL_ON_CLEANUP_ERR' - fail in case of cleanup error,
+             *  'NO_CLEANUP' - do not cleanup, just delete,
+             *  'FORCE_DELETE' - try cleanup first but delete server anyway
+	        in case of error
+----
+
+Sometimes a new registration of a deleted (unregistered) client might not be possible.
+To solve this issue, some Salt cache files should be deleted on the {productname} Server (Salt master) before trying to re-register again:
+
+----
+rm /var/cache/salt/master/thin/version
+rm /var/cache/salt/master/thin/thin.tgz
+----
```

Comment:
same remark as on the Head PR: maybe this is too prominent as in the majority of the cases thin is not used anymore.

Reply:
thanks for the reminder, I add 4.3 here: https://github.com/uyuni-project/uyuni-docs/issues/2076